src/java/org/apache/cassandra/index/sai/disk/vector/JVectorLuceneOnDiskGraph.java

```diff
@@ -51,19 +51,21 @@ protected JVectorLuceneOnDiskGraph(SegmentMetadata.ComponentMetadataMap componen
     public abstract int size();
 
     public abstract OrdinalsView getOrdinalsView() throws IOException;
+    public abstract RowIdsView getRowIdsView() throws IOException;
+    public abstract boolean isOrdinalsToRowIdsIdentityMapping();
```

Comment:
I don't love that we have to special case this, can we encapsulate the logic better somewhere else?